packages/forms-web-app/src/controllers/eligibility/granted-or-refused-permission.js

```diff
@@ -30,6 +37,12 @@ exports.postGrantedOrRefusedPermission = async (req, res) => {
     return;
   }
 
+  let isPlanningPermissionRefused = null;
+
+  if(validHouseholderPlanningPermissionStatusOptions.includes(planningPermissionStatus)){
+    isPlanningPermissionRefused = planningPermissionStatus.toLowerCase() === 'refused';
```

Comment:
I would suggest that we define these 3 status options in a constants file because even though they are being referenced from the validator, we've started to need to hard-code them in other places.

The validator can also reference the constants file.

For example https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/business-rules/src/constants.js